src/main/target/common_fc_pre.h

```diff
@@ -76,6 +76,12 @@
 #define DEFAULT_AUX_CHANNEL_COUNT       6
 #endif
 
+#ifdef USE_ITCM_RAM
+#define CRITICAL_SECTION __attribute__((section(".critical_code")))
```

Comment:
`CRITICAL_SECTION` is very confusing name (https://en.wikipedia.org/wiki/Critical_section)